src/data/faq.json

```
@@ -627,7 +644,7 @@
                             "The procedure is based on the existing CWA warning process: If an event participant or a visitor of a location has tested positive, he/she should share this result via the CWA. As a result, all other participants who were checked in at the same time will receive an immediate warning without having to burden the health authorities. Depending on the epidemiological situation, the warning is displayed in one of two versions: 'low risk' (green tile) or 'high risk' (red tile).",
                             "<b>How is the risk determined?<br>When do I receive a low-risk warning (green tile), when do I receive a high-risk warning (red tile)? </b>",
                             "The calculation of the risk depends on whether and for how long a user's attendance overlaps with the attendance of a person who later tested positive about COVID-19. If the attendance overlaps by less than 10 minutes, users receive a warning for a low-risk encounter (green tile). If a user's attendance overlaps by 10 minutes or more with that of a person who later tested positive for COVID-19, users will receive a warning about an encounter with increased risk (red tile).",
-                            "<b>Can the user see what the warning is based on, i.e. which scanned Event QR code caused the warning?</b>",
+                            "<b>Can the user see what the warning is based on, i.&nbsp;e. which scanned Event QR code caused the warning?</b>",

```

```suggestion
                            "<b>Can the user see what the warning is based on, i.e. which scanned Event QR code caused the warning?</b>",
```
Remove `&nbsp;` from English abbreviation.